elyra/pipeline/kfp/processor_kfp.py

```diff
@@ -1058,11 +1059,21 @@ def _compose_container_command_args(
         account whether the container will run in a CRI-O environment.
         """
         elyra_github_org = os.getenv("ELYRA_GITHUB_ORG", "elyra-ai")
```

Comment:
We should change the default org to `opendatahub-io` else we'll need to check the URL exists for `elyra-requirements.txt` and `pip.conf` as well. Wdyt @harshad16

Reply:
Sounds good, lets directly point to opendatahub-io and shift this away from elyra-ai.